FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -238,14 +263,16 @@ public TableQueryObject createTableQueryObject(DatabaseClient dbClient, List<Str
         String columns = String.join(", ", columnNames);
         String statementString;
         if (filters.isEmpty()) {
-            statementString = String.format("SELECT %s FROM Main_Assets ORDER By %s", columns,
+            statementString = String.format("SELECT %s FROM Main_Assets WHERE workspaceId = @workspaceId ORDER By %s", columns,
                     columnNames.get(0));
         } else {
             List<String> queryFilters = new ArrayList<>();
             for (String filter : filters) {
                 String oneFilter = String.format("%s = @%s", filter, filter);
                 queryFilters.add(oneFilter);
             }
+            String workspaceIdFilter = String.format("%s = @%s", "workspaceId", "workspaceId");
```

Comment:
why not `String workspaceIdFilter = "workspaceId= @workspaceId"` ?

Reply:
done